320px.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>All I need is...</title>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="main.css">
+        <link rel="stylesheet" media="(max-width: 980px)" href="max-980px.css">
+        <link rel="stylesheet" media="(min-width: 980px)" href="min-980px.css">
+    </head>
+    <body>
+        <header>
+            <span class="header-title">Navigation</span>
+            <div class="menu">
+                <div class="button">
+                    <a href="">
+                        <div>
+                            <i class="fas fa-bars"></i>
+                        </div>
+                    </a>
+                </div>
+                <ul class="menu-content">
+                    <li>my little pony</li>
+                    <li>cookies</li>
+                    <li>cakes</li>
+                    <li>friends</li>
+                    <li>laugh</li>
+                </ul>
+            </div>
+            <ul>
+                <li>my little pony</li>
+                <li>cookies</li>
+                <li>cakes</li>
+                <li>friends</li>
+                <li>laugh</li>
+            </ul>
+        </header>
+        <main>
+            <a id="main-a" href="">
+                <div class="main-article">
```

Comment:
It's a bad idea wrap these content into a tag, try to think about refactoring this